app/src/androidTest/java/com/google/samples/apps/topeka/activity/SignInActivityTest.java

```diff
@@ -65,15 +71,33 @@ public void checkFab_initiallyNotDisplayed() {
     }
 
     @Test
+    public void signIn_checkIfEditTextViewsAreInitiallyEmpty() {
+        onView(withId(R.id.first_name)).check(matches(withText(isEmptyOrNullString())));
+        onView(withId(R.id.last_initial)).check(matches(withText(isEmptyOrNullString())));
+    }
+
+    @Test
     public void signIn_performSuccessful() {
         inputData();
-        onView(withId(R.id.done)).check(matches(isDisplayed()));
+        onView(withId(R.id.done)).check(matches(isDisplayed())).perform(click());
+        onView(withId(R.id.toolbar_player)).check(matches(isDisplayed()));
```

Comment:
You're assuming that there's an actual behavior behind the button that then displays another activity with a view that has the ID `toolbar_player`. This might not always be true. If you want to perform a check, perform an assertTrue on `PreferencesHelper. isSignedIn(...)`.